modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -42,66 +34,66 @@ Some actions are not yet supported:
 // Reason: RMT and SMT cannot mirror Debian repositories (yet) and so cannot create it in that directory the correct files.
 // We are waiting for SMT to release the feature/fix to mirror Debian repositories. When this has been done, this comment and the limitation above can be removed.
 
+== Prepare {ubuntu} Clients for Registration
 
-This section describes how to prepare {ubuntu} clients for registration to your {susemgr} Server.
+Some preparation is required before you can register {ubuntu} clients to {productname} Server.
 
-// SUSE Manager specific instructions
-//ifeval::[{suma-content} == true]
-
-.Procedure: Setting up the software channels for {ubuntu} Client registration
+Before you begin, ensure you have the {ubuntu} product enabled, and have synchronized the {ubuntu} channels:
 
-Ensure you have the corresponding {ubuntu} product enabled and required channels have been fully synchronized:
+For {ubuntu} 18.04:
 
-.Ubuntu 18.04
 * Product: {ubuntu} Client 18.04
 * Mandatory channels: [systemitem]``ubuntu-18.04-pool-amd64``
 
-.Ubuntu 16.04
+For {ubuntu} 16.04:
+
 * Product: {ubuntu} Client 16.04
 * Mandatory channels: [systemitem]``ubuntu-16.04-pool-amd64``
 
+[NOTE]
+====
+The mandatory channels do not contain {ubuntu} upstream packages.
+The repositories and channels for synchronizing upstream content must be configured manually.
+====
 
-After synchronization the following vendor channels will be created:
 
+
+// SUSE Manager specific instructions
+ifeval::[{suma-content} == true]
```

Comment:
Because of this `if` i cannot view this section. It is harder to review it. So I hope itnis ok

Reply:
just commented that out for the moment